bgpd/bgpd.h

```diff
@@ -1794,6 +1794,12 @@ extern int bgp_option_set(int);
 extern int bgp_option_unset(int);
 extern int bgp_option_check(int);
 
+/* set the bgp no-rib option during runtime and remove installed routes */
+extern int bgp_option_norib_set_runtime(void);
```

Comment:
I see there is nothing to return actually, can be converted to void or there should be actually something more useful return instead of only 0?